src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
 body {
   margin: 0;
+  background-color: rgb(116, 116, 116);
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: solid #f3f3f3 1px;
+}
+
+.cardTop {
+  display: flex;
+
+  flex-direction: column;
+  align-items: center;
+
+  &__Img {
+    background-image: url(../images/imac.jpeg);
+
+    margin: 0 19px;
```

Comment:
The `margin` property is incorrectly defined as `margin: 0 19px;`. If the intention is to set `margin-right`, it should be defined as `margin-right: 19px;`.